testsuite/src/main/java/io/netty/testsuite/transport/socket/SocketHalfClosedTest.java

```diff
@@ -479,7 +480,7 @@ private static final class AutoCloseFalseLeader extends SimpleChannelInboundHand
         private final CountDownLatch doneLatch;
         private final AtomicReference<Throwable> causeRef;
         private int bytesRead;
-        private boolean seenOutputShutdown;
+        boolean seenOutputShutdown;
```

Comment:
@chrisvest I think I missed this in the other review but shouldn't this be volatile as we access it from the outside.
```suggestion
        volatile boolean seenOutputShutdown;
```

Reply:
It piggy-backs on `doneLatch`, so not a correctness issue, only clarity perhaps

Reply:
True... let's just keep it as it is